src/Sulu/Bundle/AdminBundle/Resources/js/components/ColorPicker/README.md

```diff
@@ -0,0 +1,13 @@
+The email component can be used to get a valid email from the user.
```

Comment:
`email`? :wink:

Reply:
Maybe we should also mention in these components what happens when the icon is clicked.